pyproject.toml

```diff
@@ -171,6 +172,9 @@ all = [
     "zeep"
 ]
 
+[tool.poetry.group.extras.dependencies]
+peakina = "^0.9.1"
```

Comment:
this should be in the `[tool.poetry.dev-dependencies]` group to avoid mixing up two config formats. Also, it needs to be included in the `all` extra

Reply:
[`377de19` (#838)](https://github.com/ToucanToco/toucan-connectors/pull/838/commits/377de191e4a897997bd3babd18a43340e6a742e0)